### DISCUSSION TITLE
User column [Apr 29]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5604] [Discord Threads]
> 🙋‍♂️ **Shrey**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### In our roadmap, not supported currently.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1090814640105213983)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Reply by Deivas** [reply]
That's a needed feature indeed. Imagine running a content calendar, for example, where users could be assigned to items.
